extension/src/recorder.ts

```diff
@@ -0,0 +1,61 @@
+import { BrowserEvent } from '@/schemas/recording'
+import { runtime } from 'webextension-polyfill'
+import { generateSelector } from './selectors'
+
+// Technically we can't rely on the button index because a user could re-assign
+// the meaning of the button but it's a fringe case.
```

Comment:
It's probably pretty common among left-handed users, so I'm not sure it's a fringe case. According to MDN it doesn't change the meaning on the button though, so we should be safe:
> Users may change the configuration of buttons on their pointing device so that if an event's button property is zero, it may not have been caused by the button that is physically left–most on the pointing device; _however, it should behave as if the left button was clicked in the standard button layout._